gufe/ligandnetwork.py

```diff
@@ -14,14 +14,17 @@
 
 
 class LigandNetwork(GufeTokenizable):
-    """A directed graph connecting many ligands according to their atom mapping
+    """A directed graph connecting ligands according to their atom mapping.
+       A network can be defined by specifying only edges, in which case the nodes are implicitly added.
+      
 
     Parameters
     ----------
     edges : Iterable[LigandAtomMapping]
-        edges for this network
+        Edges for this network, each specified as a LigandAtomMapping between two nodes.
     nodes : Iterable[SmallMoleculeComponent]
-        nodes for this network
+        Nodes for this network. Any nodes already included as a part of the 'edges' will be ignored.
+        Nodes not already included in 'edges' will be added as isolated, unconnected nodes.
```

Comment:
It comes to mind that we don't have a mechanism to handle the reverse case - i.e. what if we just wanted unconnected nodes for some reason or another? This is a conversation for another time.

Reply:
opened issue here: https://github.com/OpenFreeEnergy/gufe/pull/390#discussion_r1833792370